### PR TITLE
Update the automotive page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update the automotive page

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/automotive
- Check that the content matches [the copy doc](https://docs.google.com/document/d/1PVliB92h4ZqexaZV9F5MIcX7aRG6yBAGH1hG7x8UZTw/edit#)
- Check it matches [the design](https://user-images.githubusercontent.com/65166314/87928171-47f1b600-ca7c-11ea-9cc4-45bba031ed84.png)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2946

### CODE REVIEW

File: templates/automotive/index.html [hunk 1]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+      <p class="p-heading--two">Trusted by executives. Loved by gearheads.</p>
+      <p>The <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf" class="p-link--external">best developers choose Ubuntu</a>.</p>
+      <p>The biggest pool of talent. The broadest range of tools.  The community that counts.</p>
+      <p>One company stands behind it all &mdash; Canonical.</p>
+    </div>
+    <div class="col-6">
+      <h2 class="p-heading--five">Domain experts</h2>
+      <p class="p-heading--two">Innovate in automotive with Canonical</p>
+      <p>The experts in open source operations, from cloud to desktop to device. The publishers of Ubuntu.</p>
+      <p>Get to market faster.</p>
```
Comment (pmahnke) [inline templates/automotive/index.html:398]: these three paragraphs are ticked bullets in the design

File: templates/automotive/index.html [hunk 2]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="p-heading--five">Software strategy acceleration</h2>
+      <p class="p-heading--two">Trusted by executives. Loved by gearheads.</p>
+      <p>The <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf" class="p-link--external">best developers choose Ubuntu</a>.</p>
```
Comment (pmahnke) [inline templates/automotive/index.html:390]: these three paragraphs are ticked bullets in the design

File: templates/automotive/index.html [hunk 3]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--five">Open source licensing</h2>
+      <p class="p-heading--two">Straightforward compliance</p>
+      <p>Confidently integrate open source components into your proprietary solutions.</p>
+      <p>Source identification for all software components, all the time.</p>
+      <p>Simplify compliance. Streamline certification. Protect innovation.</p>
+    </div>
+    <div class="col-6">
```
Comment (pmahnke) [inline templates/automotive/index.html:365]: all four of the images in this section are far bigger than the design

File: templates/automotive/index.html [hunk 4]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="p-heading--five">ARM and X86</h2>
```
Comment (pmahnke) [inline templates/automotive/index.html:286]: you are missing the arm/nvidia/intel logos

File: templates/automotive/index.html [hunk 5]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align-center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/0be66895-Automotive-car-sensor.svg",
```
Comment (pmahnke) [inline templates/automotive/index.html:239]: the image in the design has grey tyres?  also it is bigger in the design?

File: templates/automotive/index.html [hunk 6]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+    <div class="col-6">
+      <h2 class="p-heading--five">Over-the-air updates</h2>
+      <p class="p-heading--two">OTA? Compressed. Transactional. Signed. Simple.</p>
+      <p>Update any system, any device, any day. Future-proof your fleet.</p>
+      <p>Full OS containerisation means transactional, atomic updates. Data snapshot backup and restore. Signed software integrity. <a href="/engage/snap-deltas">Compressed delta updates</a>.</p>
+      <p><a href="/engage/snapd-whitepaper">Complete control</a>. No delayed patches, no partial updates, no unauthorised ECU software.</p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
```
Comment (pmahnke) [inline templates/automotive/index.html:222]: the design image has a dotted circle around it?

File: templates/automotive/index.html [hunk 7]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2 class="p-heading--five">In-cabin digital</h2>
+      <p class="p-heading--two">Next level infotainment</p>
+      <p>Consumer expectations are relentless.</p>
+      <p>New insights. New apps. New content. New experiences. Personalised. Contextual.</p>
+      <p>The ever-fresh experience needs secure, reliable, over-the-air updates and <a href="https://snapcraft.io/store" class="p-link--external"></a>app stores.</p>
```
Comment (pmahnke) [inline templates/automotive/index.html:205]: the link is malformed, move the </a>

File: templates/automotive/index.html [hunk 8]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+      <h2 class="p-heading--five">Scania presents</h2>
+      <p class="p-heading--two">Fast track CAD/CAM/CAE with dynamic HPC</p>
+      <p>Scania deploys agile HPC clusters for rapid digital design, validation, crash testing, and aerodynamics.</p>
+      <p>Supercomputers on-demand for CAD, CAM and CAE. Tame SLURM and get more from your cluster.</p>
+      <p><a href="https://www.brighttalk.com/webcast/6793/416900/mastering-multicloud-with-slurm-and-juju" class="p-link--external">Watch the video&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/1c2a3c27-Automotive-Masters-vid-screen.png",
```
Comment (pmahnke) [inline templates/automotive/index.html:176]: this is meant to be a video?

File: templates/automotive/index.html [hunk 9]
```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
...
+      <p class="p-heading--two">Not just the car. The whole business of automotive.</p>
+      <p>Software shapes success in automotive today.</p>
+      <p>It’s how you sim and virtually test.</p>
+      <p>It’s how you automate factories.</p>
+      <p>It’s how you operate online services.</p>
+      <p>It’s how you turn metal into experience.</p>
+      <p><a class="js-invoke-modal" href="/internet-of-things/contact-us?product=automotive">Let’s talk open source&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
```
Comment (pmahnke) [inline templates/automotive/index.html:81]: this image is wrong?

File: templates/automotive/index.html [hunk 10]
```diff
@@ -29,163 +28,9 @@ <h1>Secure Linux for the automotive sector</h1>
...
-  </div>
-</section>
-
-<section class="p-strip--light">
-  <div class="u-fixed-width">
-    <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HAVE CHOSEN UBUNTU</h2>
+    <h2 class="p-muted-heading u-align--center u-sv3">Accelerating on Ubuntu</h2>
     <ul class="p-inline-list u-align--center u-no-margin--bottom">
       <li class="p-inline-list__item">
         <div class="u-inline--child">
```
Comment (pmahnke) [inline templates/automotive/index.html:36]: the logos are all wrong

File: templates/automotive/index.html [hunk 11]
```diff
@@ -1,26 +1,25 @@
...
+      <h1>The future is <br />beyond electric. <br />It’s electrifying.</h1>
+      <p>Smarter autopilots, infotainment, factories and business models.</p>
+      <p><a class="js-invoke-modal" href="/internet-of-things/contact-us?product=automotive">Let’s talk open source&nbsp;&rsaquo;</a></p>
     </div>
-    <div class="col-5 u-align--center u-vertically-center u-hide--small">
+    <div class="col-8 u-align--center u-vertically-center u-hide--small">
       {{
         image(
-          url="https://assets.ubuntu.com/v1/580f3ae7-automotive.svg",
+          url="https://assets.ubuntu.com/v1/480fd985-Automotive-in-car-cockpit-full.svg",
```
Comment (pmahnke) [inline templates/automotive/index.html:19]: this appears to be a different image, no grey steering wheel

File: templates/automotive/index.html [hunk 12]
```diff
@@ -1,26 +1,25 @@
...
   <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Secure Linux for the automotive sector</h1>
-      <p>Leverage the most popular open source operating system to accelerate time to market for your connected vehicles. Ubuntu is the choice platform to develop cockpit infotainment, edge connectivity, mixed-criticality and AI for automotive applications. What&rsquo;s more, Ubuntu Core is designed to enhance functional safety for critical embedded systems. Canonical brings 15 years of experience, enabling and supporting Linux to the automotive sector.</p>
-      <p>Tell us about your moonshot.</p>
-      <p><a href="/internet-of-things/contact-us?product=automotive" class="p-button--positive js-invoke-modal">Get in touch</a></p>
+    <div class="col-4">
+      <h1>The future is <br />beyond electric. <br />It’s electrifying.</h1>
+      <p>Smarter autopilots, infotainment, factories and business models.</p>
+      <p><a class="js-invoke-modal" href="/internet-of-things/contact-us?product=automotive">Let’s talk open source&nbsp;&rsaquo;</a></p>
```
Comment (pmahnke) [inline templates/automotive/index.html:14]: this is p-button--positive in the design

**Comment by anthonydillon** [comment]
@pmahnke This branch was only ready for a design review but ill address your comments.

File: templates/automotive/index.html (hunk 1 above)
Comment (anthonydillon) [reply templates/automotive/index.html:398]: Good spot, done

File: templates/automotive/index.html (hunk 2 above)
Comment (anthonydillon) [reply templates/automotive/index.html:390]: Done

File: templates/automotive/index.html (hunk 3 above)
Comment (anthonydillon) [reply templates/automotive/index.html:365]: Yeah, I was going to wait for the logos in the "ARM and X86" section to get the correct height.

File: templates/automotive/index.html (hunk 4 above)
Comment (anthonydillon) [reply templates/automotive/index.html:286]: Yeah, waiting on design to provide the images.

File: templates/automotive/index.html (hunk 5 above)
Comment (anthonydillon) [reply templates/automotive/index.html:239]: Waiting @meralonso to provide the images. These are the ones in the copy doc.

File: templates/automotive/index.html (hunk 6 above)
Comment (anthonydillon) [reply templates/automotive/index.html:222]: Waiting @meralonso to provide the images. These are the ones in the copy doc.

File: templates/automotive/index.html (hunk 7 above)
Comment (anthonydillon) [reply templates/automotive/index.html:205]: Good spot, thanks

File: templates/automotive/index.html (hunk 8 above)
Comment (anthonydillon) [reply templates/automotive/index.html:176]: It's linked to an image in the copy doc but I'll try add a link around the image

File: templates/automotive/index.html (hunk 9 above)
Comment (anthonydillon) [reply templates/automotive/index.html:81]: Updated

File: templates/automotive/index.html (hunk 10 above)
Comment (anthonydillon) [reply templates/automotive/index.html:36]: Yep, there were no images provided.

**Comment by meralonso** [comment]
Hey @anthonydillon I can't see the link to review this page - could you send it over, please?

File: templates/automotive/index.html (hunk 11 above)
Comment (anthonydillon) [reply templates/automotive/index.html:19]: Yep, its the one in the copy doc

File: templates/automotive/index.html (hunk 12 above)
Comment (anthonydillon) [reply templates/automotive/index.html]: Nice one :+1:

**Comment by jpmartinspt** [comment]
Demo: https://ubuntu-com-7977.demos.haus/

**Comment by anthonydillon** [comment]
@meralonso thank you for the images in the copy doc. I have updated them now.

**Comment by meralonso** [comment]
@anthonydillon thanks for this, is looking nice. I have some tweaks, let me know if they're possible.

Has this section change? There are 9 logos on the copy doc (links at the end)
Screenshot 2020-07-23 at 10 06 22

Can we center this illustration within the 6 column space?
Screenshot 2020-07-23 at 10 09 35

Same for this one, and could you please make it bigger - 230px high?
Screenshot 2020-07-23 at 10 09 40

Can you make this screen smaller? not the nicest.
Play button [here ](https://assets.ubuntu.com/v1/88e85d2b-play-icon-white.svg)
Screenshot 2020-07-23 at 10 15 39

Can you make this illustration a bit bigger (250-260px high) and center it within the 6 column space?
Screenshot 2020-07-23 at 10 17 36

Logo order: arm -nvidia - intel. Could you make intel smaller and align them to the bottom of the icon beside?
Screenshot 2020-07-23 at 10 20 44

Let me know if you want to catch up on animations - thanks so much!

**Comment by anthonydillon** [comment]
> Has this section changed? There are 9 logos on the copy doc (links at the end)

Could you provide SVG versions on the logos, please?

> Can we center this illustration within the 6 column space?

Done

> Same for this one, and could you please make it bigger - 230px high?

Done

> Can you make this screen smaller? not the nicest.
Play button here 

Done

> Can you make this illustration a bit bigger (250-260px high) and center it within the 6 column space?

Done

> Logo order: arm -nvidia - intel. Could you make intel smaller and align them to the bottom of the icon beside?

I think I've done it. I changes the order and made the intel logo slightly smaller.

**Comment by meralonso** [comment]
cool, thanks @anthonydillon just 3 more changes:

Play button a bit bigger please, around 80px.

On this section please replace image with [this one](https://assets.ubuntu.com/v1/2bd31151-Automotive-in-car-cockpit-sized.svg), it's a different crop - 250px high
Screenshot 2020-07-23 at 14 30 43
 
Could you push the logos row to the left so it's aligned to the text? and also align the row to the bottom of the illustration on the column beside so teh text block is at the same level?
Screenshot 2020-07-23 at 14 34 24

Cheers!

**Comment by meralonso** [comment]
Links to logos below - sorry i couldn't.find toyoya and volkswagen svg version

[Toyota](https://assets.ubuntu.com/v1/4f0688c6-Toyota_Logo_silver.png)
[BMW](https://assets.ubuntu.com/v1/1e265e27-bmw-new-2020.svg)
[Volkswagen](https://assets.ubuntu.com/v1/578dbfb5-Volkswagen_logo_2019.svg.png)
[Ford](https://assets.ubuntu.com/v1/9a0de5dd-Ford-Logo.svg)
[Scania](https://assets.ubuntu.com/v1/9320c145-Scania_Logo-Symbol.svg)
[Bosch](https://manager.assets.ubuntu.com/create)
[Continental](https://assets.ubuntu.com/v1/d1da96af-Continental-Logo.svg)
[Uber](https://assets.ubuntu.com/v1/547ef405-uber.svg)
[Lyft ](https://assets.ubuntu.com/v1/4a1c3691-lyft.svg)

**Comment by spencerbygraves** [comment]
Thanks @anthonydillon just two things please.

Can we align these logos left?

image

And can we make these 2 logos 120px please

image

Thanks.

**Comment by anthonydillon** [comment]
@spencerbygraves thanks, done.

**Comment by spencerbygraves** [comment]
@anthonydillon thanks 👍

**Pull request merged**